### PR TITLE
Add missing build information

### PR DESCRIPTION
**Pull request by ahmedalsudani**
- refer to local jshint
- add electron-packager to package.json
- Maven instructions

### CODE REVIEW

File: cfg/electron-remote-components-lnx-32.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "ForceStable": false,
   "LatestRolloutPercent": 50,
-  "InstallerElectronVersion": "2015.12.30.12.39",
+  "InstallerElectronVersion": "2016.1.4.15.11",
```
Comment (ahmedalsudani) [inline cfg/electron-remote-components-lnx-32.json]: @fjvallarino @tejohnso should this be included in the PR? The change was auto-generated

**Comment by ahmedalsudani** [comment]
`version.json` was also changed automatically by the build script. That change broke the tests though. Should I include it?

**Comment by tejohnso** [comment]
@ahmedalsudani We don't normally run build.js locally and push the result.  That's for the CI server.

**Comment by tejohnso** [comment]
:+1:

**Comment by ahmedalsudani** [comment]
The last push was only resolving merge conflicts

**Pull request merged**